session_srtcp.go

```diff
@@ -112,25 +112,47 @@ func (s *SessionSRTCP) decrypt(buf []byte) error {
 		return err
 	}
 
-	pkt, err := rtcp.Unmarshal(decrypted)
-	if err != nil {
-		return err
-	}
-
-	for _, ssrc := range pkt.DestinationSSRC() {
+	forward := func(ssrc uint32) (closed bool, err error) {
```

Comment:
Don't think pulling this into a function adds clarity. It's confusing what `return true, nil` signifies, for instance.

It's only called once so it would probably be better inline like before.

Reply:
fair, i'm reverting that.